blog.html

```diff
@@ -1,291 +1,302 @@
+                                <span aria-hidden="true">
+                                    <span class="lnr lnr-chevron-left"></span>
+                                </span>
+                            </a>
+                        </li>
+                        <li class="page-item"><a href="#" class="page-link">01</a></li>
+                        <li class="page-item active"><a href="#" class="page-link">02</a></li>
+                        <li class="page-item"><a href="#" class="page-link">03</a></li>
+                        <li class="page-item"><a href="#" class="page-link">04</a></li>
+                        <li class="page-item"><a href="#" class="page-link">09</a></li>
+                        <li class="page-item">
+                            <a href="#" class="page-link" aria-label="Next">
+                                <span aria-hidden="true">
+                                    <span class="lnr lnr-chevron-right"></span>
+                                </span>
+                            </a>
+                        </li>
+                    </ul>
+                </nav> -->
+                </div>
+            </div>
+            <div class="col-lg-4">
+                <div class="blog_right_sidebar">
+                    <aside class="single_sidebar_widget popular_post_widget">
+                        <h3 class="widget_title">Popular Posts</h3>
+                        <div class="media post_item">
+                            <img src="blog/img/popular-post/post1.jpg" alt="post">
+                            <div class="media-body">
+                                <a href="post1.html"><h3>An Introduction to Arduino</h3></a>
+                            </div>
+                        </div>
+                        <div class="media post_item">
+                            <img src="blog/img/popular-post/post2.jpg" alt="post">
+                            <div class="media-body">
+                                <a href="post2.html"><h3>An Introduction to Raspberry Pi</h3></a>
+                            </div>
+                        </div>
+                        <div class="media post_item">
+                            <img src="blog/img/popular-post/post3.jpg" alt="post">
+                            <div class="media-body">
+                                <a href="post3.html"><h3>Practical Approach to Web development</h3></a>
+                            </div>
+                        </div>
+                        <div class="media post_item">
+                            <img src="blog/img/popular-post/post4.jpg" alt="post">
+                            <div class="media-body">
+                                <a href="post4.html"><h3>Best Coding Challenges Websites</h3></a>
+                            </div>
+                        </div>
+                        <div class="br"></div>
+                    </aside>
+                </div>
+            </div>

```

Comment:
@ankitthakur01 The size of this section is not as per the page. Style font of the page where you are styling body in blog.css